pyxem/signals/pair_distribution_function1d.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017-2019 The pyXem developers
+#
+# This file is part of pyXem.
+#
+# pyXem is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# pyXem is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with pyXem.  If not, see <http://www.gnu.org/licenses/>.
+"""Signal class for Pair Distribution Function (PDF) radial profiles
+as a function of distance r.
+"""
+import numpy as np
+from hyperspy.signals import Signal1D
+
+from pyxem.utils.pdf_utils import normalise_pdf_signal_to_max
+
+
+class PairDistributionFunction1D(Signal1D):
+    _signal_type = "pair_distribution_function1d"
+
+    def __init__(self, *args, **kwargs):
+        Signal1D.__init__(self, *args, **kwargs)
+
+    def normalise_signal(self, s_min=0, inplace=False, *args, **kwargs):
+        """
+        Normalises the Reduced PDF signal to having a maximum of 1.
+        This is applied to each pdf separately in a multidimensional signal.
+
+        Parameters
+        ----------
+        s_min : float
+            The minimum scattering vector s to be considered. Values at lower s
+            are ignored. This is to prevent the effect of large oscillations at
+            r=0. If not set, the full signal is used. Normalisation can aid
+            unsupervised machine learning methods distinguish signals present.
```

Comment:
Try not to add random unrelated opinions/comments/advice to docstrings. So remove the thing about Normalisation affecting machine learning.